tgbotapi.core/src/commonMain/kotlin/dev/inmo/tgbotapi/types/chat/ExtendedAbstracts.kt

```diff
@@ -20,10 +20,7 @@ sealed interface ExtendedChat : Chat {
     val profileBackgroundCustomEmojiId: CustomEmojiId?
     val maxReactionsCount: Int
 
-    /**
-     * Represent `can_send_gifts` field
-     */
-    val canReceiveGifts: Boolean
```

Comment:
This field must be kept for backward compatibility with getter using new `acceptedGiftTypes`